piccolo/apps/asgi/commands/new.py

```diff
@@ -15,7 +15,7 @@
     "blacksheep": ["blacksheep"],
     "litestar": ["litestar"],
     "esmerald": ["esmerald"],
-    "lilya": ["lilya"],
+    "lilya": ["lilya", "orjson"],
```

Comment:
Interesting that `lilya` would do this without having it as one of their own dependencies, but I'm OK with adding this if it works. 👍

Reply:
Reverted this because [Lilya now uses standard library json and no longer orjson](https://github.com/dymmond/lilya/releases/tag/0.18.1).